recipe/bld.bat

```diff
@@ -0,0 +1,20 @@
+setlocal EnableDelayedExpansion
+
+cmake -B build ^
+  -G "Ninja" ^
+  -DSUITESPARSE_ENABLE_PROJECTS="suitesparse_config;amd;btf;camd;ccolamd;colamd;cholmod;cxsparse;ldl;klu;umfpack;paru;rbio;spqr" ^
+  -DBLA_VENDOR=OpenBLAS ^
+  -DBUILD_SHARED_LIBS=ON ^
+  -DBUILD_STATIC_LIBS=OFF ^
+  -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON ^
```

Comment:
```suggestion
```

This is already done by suitesparse's CMake, let's remove it so it they switch to some other method to deal visibility the recipe will not interfere with that.